app/build.gradle

```diff
@@ -8,12 +8,6 @@ android {
             storeFile file('/home/wyxiang/Android/jks/hitschedule.jks')
             storePassword 'hitschedule'
         }
-        debug {
```

Comment:
I delete these lines because I don't have the maintainer's keystore.  You can write it again on your computer.